build.sbt

```diff
@@ -46,11 +46,7 @@ lazy val framework =
       case _ =>
         Seq()
     }) //workaround for scala/scala-dev#249
-    /*.settings(
-      (sources / aggregate) in (Compile, doc),
-      aggregatedSetting(dependencyClasspath in(Compile, doc)),
-      publishArtifact := false
-    )*/
+    .enablePlugins(ScalaUnidocPlugin)
```

Comment:
Just to check---is this still officially a workaround for scala/scala-dev#249 per the comment above it? Happy to make this edit.

Reply:
Oh. To be clear this comment about the Scala bug is for the block above, not the block below 😬

Reply:
These lines are about the compiled mega docs. It’s currently misbehaving but I think this PR is still a win.